rundeck-scripts/start-node

```diff
@@ -167,6 +167,13 @@ while (( i )); do
 	if [[ -n "$container_id" ]]; then
 		echo "RNode is running"
 		nohup docker logs -f $container_id &> $DIAG_DIR/console.log &
+
+                node_pid="$(docker inspect -f '{{.State.Pid}}' rnode || echo 0)"
+                if (( node_pid == 0 )); then
+                        echo "Could not get node PID"
+                        break
+                fi
+                nohup ./pmap.py "$node_pid" "perf-bootstrap.c.developer-222401.internal" 8091 testnet1 &
```

Comment:
This script is being run by basically `ssh foo /opt/rchain-testnet-node/rundeck-script/start-node` so current working directory is not `/opt/rchain-testnet-node`. You should replace `./pmap.py` with `$INSTALL_DIR/pmap.py`.